CHANGELOG.md

```diff
@@ -1,43 +1,94 @@
-- Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>)
+- Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>) (#176)
+- Add hint in footer of how to copy text, if attempt to select text (with mouse) in terminal
+- Stream search is now case-insensitive
+- Add command-line option to check version (#280)
 - Allow <kbd>f</kbd> and <kbd>P</kbd> shortcut keys to work from side panels (narrow starred & private messages)
 - Warn on startup, if specified theme is lacking current required styles
 - Upon unexpected crash, exit cleanly and log traceback to `zulip-terminal-tracebacks.log`
-- Add <kbd>z</kbd> hot/shortcut key to toggle between stream/topic or all-private/specific-person narrows
 
 ### Visual improvements
 - Right-align unread-counts in left & right panels (as in webapp)
 - Remove intrusive flashing cursor in side panels
-- Truncate text in left & right panels cleanly with '..', avoiding text overflow
-- Show user status as `active` (Green), `idle` (Yellow) or `offline` (White) using different colors.
+- Truncate text in left & right panels cleanly with '…', avoiding text overflow (#237)
+- Show users' status as `active` (Green), `idle` (Yellow) or `offline` (White) using different colors.
+- Update user list every minute, not just at startup (#330)
+- Sort user list alphabetically, case-independently (#264)
 - Further improvement/reordering of shortcut keys in README & help menu (<kbd>?</kbd>)
-- Improve styling of help menu, and how the menu scales with application width & height
+- Improve styling of help menu, and how it menu scales with application width & height (#297, #307)
 - Make stream icons bold and correct background color
+- Themes improved (#254)
+- Support terminal transparency in default theme (#377)
+- Clarify that <kbd>ctrl</kbd>+<kbd>c</kbd> is the intended way to exit/quit (#208)
+- Mark edited messages with 'EDITED' at their top-left
+- Show current narrow with search box, above current-recipient indicator for message (#201, #340)
+- Hide current-recipient indicator when fully narrowed (in topic or conversation)
+- Improve rendering of quoted text with indents
+- Display custom emoji text-equivalents in messages and reactions (#238)
+- Readline keys are now listed in the online help
+- Change main widget styles, to clearly separate three columns, with bold top-lines
+- Minimize space taken up by search boxes & display shortcut keys for each
+- Remove bullets & display shortcut keys for for all messages, private messages & starred messages
+- Style private message header like webapp (You and ...)
 
 ### Important bugfixes
-- Fix bug potentially mixing unread counts for messages from users & streams
-- Don't increase the unread counts if we sent the message
-- Exit cleanly if cannot connect to zulip server
-- Avoid crash in rare case of empty message content
+- Bug where new messages from server would not be shown in message list
```

Comment:
Should we also add `Bug when message sent was not appended in streams. (#362, #364, #365)

Reply:
I think this line was attempting to include those issues, broadly. Note that the other numbers I gave should all have been issues, not PRs - otherwise it would be a lot more busy with numbers/ids and take even more time.